vanhouse/src/components/UserList.js

```diff
@@ -32,11 +49,9 @@ function UserList({ setQuery, setUserId }) {
                 const newName = [];
                 console.log(res);
                 const cardList = res.map((item, index) => (
-                    <div className="user-item" key={item} >
+                    <div className="user-item" key={item}  onClick={(e) => filterUserPost(e, item)}>
```

Comment:
@tyxxxx  Let's use `item._id` as the key, otherwise it shows errors in the console.